app/docs/0.10.x/plugin-development/distribution.md

```diff
@@ -1,74 +1,300 @@
 ---
-title: Plugin Development - Distribute your plugin
+title: Plugin Development - (un)install your plugin
```

Comment:
I think `Distribute` is a better title, since you document more than installing and uninstalling instructions.